make-util.js

```diff
@@ -915,7 +941,7 @@ var createAggregatedZip = function (packagePath) {
         });
 
     // validate task uniqueness within the layout based on task GUID + major version
-    var majorVersions = { };
+    var majorVersions = {};
```

Comment:
why change the whitespace multiple places in the code where you are not making any changes?